calabash-cucumber/lib/calabash-cucumber/usage_tracker.rb

```diff
@@ -131,26 +147,41 @@ def used_cucumber?
       #
       # Collect a hash of usage info.
       def info
-        {
+
+        allowed = allowed_to_track
+
+        return {} if allowed == "none"
+
+        # Events only
+        hash = {
           :event_name => "session",
```

Comment:
> To which does "session" refer?

None. 

There are many things we _might_ want to track.  At the moment, we are only doing a head count with some additional info.

We are tracking IRB (console) vs cucumber.

Later, once we've deployed this first version, we can analyze what other information we might collect.